build/compiler.py

```diff
@@ -353,10 +353,16 @@ def lint(self, fix=False, force=False):
 
     Returns:
       True on success; False on failure.
-    """
+    """  
     deps = self.source_files + [self.config_path]
     if not force and not _must_build(self.output, deps):
       return True
+    
+    """Windows shows an error when the file location has '\\' so for windows the below if condition changes the'\\' with '/' in variable storing file locations """
+    if os.name == 'nt' :
+      self.config_path = self.config_path.replace('\\','/')
+      for j in range(0,len(self.source_files)):  
+        self.source_files[j] = self.source_files[j].replace('\\','/')
```

Comment:
A more Python way to do this  would be:

```py
self.source_files = [f.replace('\\', '/') for f in self.source_files]
```

Reply:
That's a cool way . Concating for loop and statement into a single line . Now I'll always use that from now . Just a follow up , can we add more than one line of code in that format .

Reply:
I'm not sure what you mean.  This is called list-comprehension.  This is just assigning to a new array it just created.  The basic format is `[val for f in list]` where `val` can be any expression and is each element's value and `list` is the source list.